### DISCUSSION TITLE
Tero Subtitler 1.0.1.7

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#278] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.7'>Tero Subtitler 1.0.1.7</a>.</em>

**Comment by AK-Nobleman** [comment]
Can anyone help me with this?
it says error creating registry key :
HKEY_CLASSES_ROOT\.ass

RegCreateKeyEx failed; code 5
Access is denied

I've already tried disabling my antivirus but it still shows that, I've also tried to download an older version but the same error shows

**Reply by URUWorks** [reply]
Possibly you are using a user who does not have permissions to modify the registry?
